### ISSUE TITLE
[GitHub] Provide ability to switch between branches

### ISSUE THREAD

**Issue by krnekhelesh** [sonrisesoftware/project-dashboard#100]
As people start using Project Dashboard more, they try to perform more git related actions using the app. One common task that every developer does is switch branches during development. It would be nice if the app displays a list of the branches (local and remote) and provide the ability to switch between them.

**Comment by iBelieve** [comment]
So you're saying to tell the app where you have the code checked out and let you switch branches and create and push commits?

**Comment by krnekhelesh** [comment]
yeah something like that. In the phone mode, this feature may not be that useful considering that users generally do not fork github projects on their phone :) ..but on the desktop this will be quite useful. And you will not be breaking the security enforcements since you will be only reading a folder or file and not writing to them.

**Comment by iBelieve** [comment]
Yeah, that would be very useful.

And even on the phone, it would be nice to be able to browse the source code in the remote repository, similar to issue #99.
